docs/inheritance.md

```diff
@@ -0,0 +1,100 @@
+# Inheritance
+
+An `OpenStackDataPlaneNode` inherits any attribute of an
+`OpenStackDataPlaneRole` but those attributes my also be overridden on
```

Comment:
nit: `OpenStackDataPlaneRole` but those attributes may also be overridden on

Reply:
Thanks, fixed.